### PR TITLE
Kucerar/3150 other account request

### PR DESCRIPTION
**Pull request by kucera-roman**
https://redmine.czechidm.com/issues/3150

### CODE REVIEW

File: Realization/frontend/czechidm-acc/src/content/wizard/AccountWizardDetail.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            show={!!show}/>
+        </IdmContext.Provider>
+      </Basic.Div>
+    );
+  }
+
+}
+
+AccountWizardDetail.defaultProps = {
+  reopened: false // Defines if the wizard use for create new system or for reopen existed.
```
Comment (tomdoischer) [inline Realization/frontend/czechidm-acc/src/content/wizard/AccountWizardDetail.js:49]: account instead of system in the comment

**Review by tomdoischer — CHANGES_REQUESTED** [review]
The code looks good, minor things in the comments.

During testing, I found that:

- for other personal accounts, the identity account is not created
- the account agenda doesn't show the system entity for some reason for accounts created in the wizard
- [minor, can ignore] if I create an account with a uid that is already used, I get a pretty generic error message, I would like to know that the uid is used

File: Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/rest/impl/AccAccountController.java [hunk 2]
```diff
@@ -612,7 +620,76 @@ protected AccAccountFilter toFilter(MultiValueMap<String, Object> parameters) {
...
+			response = AccountWizardDto.class,
+			tags = { AccAccountController.TAG },
+			authorizations = {
+					@Authorization(value = SwaggerConfig.AUTHENTICATION_BASIC, scopes = {
+							@AuthorizationScope(scope = AccGroupPermission.ACCOUNT_READ, description = "")}),
+					@Authorization(value = SwaggerConfig.AUTHENTICATION_CIDMST, scopes = {
+							@AuthorizationScope(scope = AccGroupPermission.ACCOUNT_READ, description = "")})
+			})
+	public ResponseEntity<AccountWizardDto> loadWizardType(@NotNull @Valid @RequestBody AccountWizardDto wizardDto) {
+		return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
```
[the remark this answers is not in the document]
Comment (kucera-roman) [reply Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/rest/impl/AccAccountController.java:691]: fixed

File: Realization/frontend/czechidm-acc/src/content/wizard/AccountWizardDetail.js (hunk 1 above)
Comment (kucera-roman) [reply Realization/frontend/czechidm-acc/src/content/wizard/AccountWizardDetail.js]: fixed

**Comment by kucera-roman** [comment]
I added new exception if no login is found.
This will prevent issue when broken account could be created.

For account created via wizard I can see everything in account agenda. We can look at it together

**Comment by tomdoischer** [comment]
Looks good to me!

**Pull request merged**